scalability_and_performance/ztp_far_edge/ztp-sno-additional-worker-node.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="ztp-sno-additional-worker-node"]
+= Additional worker nodes in a {sno} cluster
+include::_attributes/common-attributes.adoc[]
+:context: sno-additional-worker
+
+toc::[]
+
+You can add one or more worker nodes to an existing {sno-caps} (SNO) cluster to increase CPU resources used by the original SNO control plane node. The addition of worker nodes does not require any downtime for the existing SNO.
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about {sno} clusters tuned for vDU application deployments, see xref:../../scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc#sno-configure-for-vdu[Reference configuration for deploying vDUs on {sno}].
+
```

Comment:
I think you could also add an xref to https://docs.openshift.com/container-platform/4.11/nodes/nodes/nodes-sno-worker-nodes.html as well.